### PR TITLE
[Fix] fix pytorch-paddle mapping index generate

### PR DESCRIPTION
**Pull request by RedContritio**

### CODE REVIEW

**Comment by RedContritio** [comment]
@sunzhongkai588 请看[预览](http://preview-pr-6463.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/zh/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.html)，业已修复

**Review by luotao1 — COMMENTED** [review]
要不要把新增的几篇md文档去掉？这些需要 @zhwesky2010 review 吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LogSigmoid.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+torch.nn.LogSigmoid(*args, **kwargs)
+```
+
+### [paddle.nn.LogSigmoid](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/LogSigmoid_cn.html#logsigmoid)
+
+```python
+paddle.nn.LogSigmoid(name=None)
+```
+
+两者功能一致，参数完全一致。
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LogSigmoid.md]: 这个是不是torch可以输入参数，paddle不能输入。torch参数更多？

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Sigmoid.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## [ 参数完全一致 ] torch.nn.Sigmoid
+
+### [torch.nn.Sigmoid](https://pytorch.org/docs/stable/generated/torch.nn.Sigmoid.html)
+
+```python
+torch.nn.Sigmoid(*args, **kwargs)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Sigmoid.md:6]: 这个是不是torch可以输入参数，paddle不能输入。torch参数更多？

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softsign.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+## [ 参数完全一致 ] torch.nn.Softsign
+
+### [torch.nn.Softsign](https://pytorch.org/docs/stable/generated/torch.nn.Softsign.html)
+
+```python
+torch.nn.Softsign(*args, **kwargs)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Softsign.md:6]: 这个是不是torch可以输入参数，paddle不能输入。torch参数更多？

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Tanh.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+## [ 参数完全一致 ] torch.nn.Tanh
+
+### [torch.nn.Tanh](https://pytorch.org/docs/stable/generated/torch.nn.Tanh.html)
+
+```python
+torch.nn.Tanh(*args, **kwargs)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Tanh.md:6]: 这个是不是torch可以输入参数，paddle不能输入。torch参数更多？

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Tanhshrink.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+## [ 参数完全一致 ] torch.nn.Tanhshrink
+
+### [torch.nn.Tanhshrink](https://pytorch.org/docs/stable/generated/torch.nn.Tanhshrink.html)
+
+```python
+torch.nn.Tanhshrink(*args, **kwargs)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Tanhshrink.md:6]: 这个是不是torch可以输入参数，paddle不能输入。torch参数更多？

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md [hunk 6]
```diff
@@ -616,326 +616,14 @@
...
 | REFERENCE-MAPPING-ITEM(`torch.nn.functional.upsample_bilinear`, https://github.com/PaddlePaddle/docs/tree/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/functional/torch.nn.functional.upsample_bilinear.md) |
-| (torch.nn.functional.multilabel_margin_loss)[https://pytorch.org/docs/stable/generated/torch.nn.functional.multilabel_margin_loss.html#torch.nn.functional.multilabel_margin_loss] | 功能缺失 |
-| (torch.nn.functional.ctc_loss)[https://pytorch.org/docs/stable/generated/torch.nn.functional.ctc_loss.html#torch.nn.functional.ctc_loss] | 功能缺失，torch 的 log_softmax+ctc_loss 相当于 paddle 的 ctc_loss |
+| NOT-IMPLEMENTED-ITEM(`torch.nn.functional.multilabel_margin_loss`, https://pytorch.org/docs/stable/generated/torch.nn.functional.multilabel_margin_loss.html#torch.nn.functional.multilabel_margin_loss) |
+| NOT-IMPLEMENTED-ITEM(`torch.nn.functional.ctc_loss`, https://pytorch.org/docs/stable/generated/torch.nn.functional.ctc_loss.html#torch.nn.functional.ctc_loss) |
 
 ***持续更新...***
 
 ## torch.Tensor.XX API 映射列表
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:624]: 这些删掉的原因是？

**Comment by zhwesky2010** [comment]
@RedContritio  /docs/docs/guides/model_convert/pytorch_api_mapping_cn.md  这个文件是不是也可以顺带删除了，有些误导性

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.vsplit.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-## [ 仅参数名不一致 ]torch.Tensor.split_size_or_sections
+## [ 仅参数名不一致 ]torch.Tensor.vsplit
```
Comment (luotao1) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.vsplit.md]: - https://github.com/PaddlePaddle/docs/pull/6464
vsplit.md和matrix_exp.md，在上面这个PR里改了，可以merge下develop，去掉这部分

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LogSigmoid.md (hunk 1 above)
Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LogSigmoid.md]: 这几个虽然 torch 可以输入参数，但是参数并不起到任何作用，我倾向于其算是无参数的情况

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md (hunk 6 above)
Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:624]: 因为上面几行是手动维护的，改成了自动维护的格式；因为其他的功能缺失没有原因描述，所以这里把原因也删除了

**Comment by RedContritio** [comment]
> @RedContritio /docs/docs/guides/model_convert/pytorch_api_mapping_cn.md 这个文件是不是也可以顺带删除了，有些误导性

好的已删除

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.vsplit.md (hunk 7 above)
Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.vsplit.md]: 已 merge

**Comment by RedContritio** [comment]
> 要不要把新增的几篇md文档去掉？这些需要 @zhwesky2010 review 吧

这部分是因为这几个 api 在目录里提及了，所以需要有对应的映射描述文件。

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md [hunk 8]
```diff
@@ -1537,7 +1230,7 @@
 | ----- | ----------- | ----------------- | --- |
 | REFERENCE-MAPPING-ITEM(`torch.linalg.svd`, https://github.com/PaddlePaddle/docs/tree/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.svd.md) |
 | REFERENCE-MAPPING-ITEM(`torch.linalg.matrix_power`, https://github.com/PaddlePaddle/docs/tree/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.matrix_power.md) |
-| REFERENCE-MAPPING-ITEM(`torch.linalg.matrix_exp`, https://github.com/PaddlePaddle/docs/tree/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.matrix_exp.md) |
+| REFERENCE-MAPPING-ITEM(`torch.linalg.matrix_power`, https://github.com/PaddlePaddle/docs/tree/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/linalg/torch.linalg.matrix_exp.md) |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md]: 后面这个链接看起来是错的，torch.linalg.matrix_exp.md？

Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md]: 已更新，列表项改回了 `torch.linalg.matrix_exp`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool3d.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
+## [ 参数完全一致 ] torch.nn.AdaptiveAvgPool3d
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool3d.md]: 这个是 paddle参数更多 吧

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md [hunk 10]
```diff
@@ -616,326 +616,14 @@
...
+| NOT-IMPLEMENTED-ITEM(`torch.nn.functional.ctc_loss`, https://pytorch.org/docs/stable/generated/torch.nn.functional.ctc_loss.html#torch.nn.functional.ctc_loss) |
 
 ***持续更新...***
 
 ## torch.Tensor.XX API 映射列表
 梳理了`torch.Tensor.XX`类 API 的 PyTorch-PaddlePaddle API 映射列表。
 
-| 序号  | PyTorch API | PaddlePaddle API | 备注 |
- |-----| --- | --- | --- |
-| 1   |  [torch.Tensor.device](https://pytorch.org/docs/stable/generated/torch.Tensor.device.html)  |  [paddle.Tensor.place](https://www.paddlepaddle.org.cn/documentation/docs/api/paddle/Tensor_cn.html#place)  |    功能一致, 无参数 , [差异对比](https://github.com/PaddlePaddle/docs/tree/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.device.md)  |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:629]: 这里删除了上百个 `torch.Tensor.*` 的原因是？

Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:629]: 这里的是之前其他人 pr 的时候，错误地将生成结果和源文件复制到了一起，产生的错误。 [相关PR](https://github.com/PaddlePaddle/docs/pull/6254)

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool3d.md (hunk 9 above)
Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool3d.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool2d.md [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+### [torch.nn.AdaptiveAvgPool2d](https://pytorch.org/docs/stable/generated/torch.nn.AdaptiveAvgPool2d.html)
+
+```python
+torch.nn.AdaptiveAvgPool2d(output_size)
+```
+
+### [paddle.nn.AdaptiveAvgPool2D](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/AdaptiveAvgPool2D_cn.html#adaptiveavgpool2d)
+
+```python
+paddle.nn.AdaptiveAvgPool2D(output_size, data_format='NCHW', name=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool2d.md:12]: 这个不是paddle参数更多？

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool3d.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+### [torch.nn.AdaptiveAvgPool3d](https://pytorch.org/docs/stable/generated/torch.nn.AdaptiveAvgPool3d.html)
+
+```python
+torch.nn.AdaptiveAvgPool3d(output_size)
+```
+
+### [paddle.nn.AdaptiveAvgPool3D](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/AdaptiveAvgPool3D_cn.html#adaptiveavgpool3d)
+
+```python
+paddle.nn.AdaptiveAvgPool3D(output_size, data_format='NCDHW', name=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool3d.md:12]: 这个不是paddle参数更多？

**Comment by zhwesky2010** [comment]
@RedContritio 写文档需要尤其细心点，可以先自己整体自查一遍

**Comment by RedContritio** [comment]
> @RedContritio 写文档需要尤其细心点，可以先自己整体自查一遍

好的了解，关于 torch 支持 `(*args, **kwargs)`  但不实际生效这种情况，目前还是定为参数完全一致，其余问题已检查。

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LogSigmoid.md [hunk 13]
```diff
@@ -0,0 +1,15 @@
+## [ 参数完全一致 ] torch.nn.LogSigmoid
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LogSigmoid.md]: 如果不生效，是不是叫 无参数 更好

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool2d.md [hunk 14]
```diff
@@ -0,0 +1,20 @@
+## [ 参数完全一致 ] torch.nn.AdaptiveAvgPool2d
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.AdaptiveAvgPool2d.md]: 仅 paddle 参数更多

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LogSigmoid.md (hunk 13 above)
Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.LogSigmoid.md]: 确实。无参数的 4 个 api 已修改

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Tanh.md [hunk 15]
```diff
@@ -0,0 +1,15 @@
+## [ 参数完全一致 ] torch.nn.Tanh
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.Tanh.md]: 这个也是 无参数 吧

**Review by zhwesky2010 — APPROVED** [review]
lgtm

建议那几个无参数的，后面把签名也改成无参数的形式，还是要以实际代码来，不用一定按文档

**Comment by RedContritio** [comment]
> lgtm
> 
> 建议那几个无参数的，后面把签名也改成无参数的形式，还是要以实际代码来，不用一定按文档

好的了解，后续做对齐和检查的时候处理这种类型的问题。
